translate/storage/test_po.py

```diff
@@ -537,64 +511,65 @@ def test_makeobsolete_plural(self):
 #~ msgstr[1] "Koeie"
 '''
         pofile = self.poparse(posource)
-        print(pofile)
         unit = pofile.units[0]
         assert not unit.isobsolete()
         unit.makeobsolete()
         assert unit.isobsolete()
-        print(pofile)
         assert str(unit) == poexpected
 
     def test_makeobsolete_msgctxt(self):
         """Tests making a unit with msgctxt obsolete"""
-        posource = '#: test.c\nmsgctxt "Context"\nmsgid "test"\nmsgstr "rest"\n'
-        poexpected = '#~ msgctxt "Context"\n#~ msgid "test"\n#~ msgstr "rest"\n'
+        posource = ('#: test.c\n'
+                    'msgctxt "Context"\n'
+                    'msgid "test"\nmsgstr "rest"\n')
+        poexpected = ('#~ msgctxt "Context"\n'
+                      '#~ msgid "test"\n#~ msgstr "rest"\n')
```

Comment:
You seem to have split this into several lines for readability. I like that. But it seems the `msgstr` in `poexpected` is not on its own line.